pages/cutout/[image_name].tsx

```diff
@@ -35,9 +35,9 @@ export default function CutoutPage() {
   }
 
   return (
-    <div className='flex items-center justify-center gap-2'>
-        {OriginalImage && Cutouts ? <CutoutPreviews singleFile={OriginalImage} multipleFiles={Cutouts} /> : <CssLoader />}  
+    <div className='flex flex-col items-center justify-center gap-2'>
         {classes && <ClassDisplay classes={classes} />}
+        <CutoutPreviews singleFile={OriginalImage} multipleFiles={Cutouts} /> 
```

Comment:
**Suggestion:** Consider checking if `OriginalImage` and `Cutouts` are not null or undefined before rendering `CutoutPreviews`. This will prevent any potential errors or unexpected behavior if these props are not provided. [possible issue]
```suggestion
        {OriginalImage && Cutouts && <CutoutPreviews singleFile={OriginalImage} multipleFiles={Cutouts} />}
```